events/open-source-marathon/index.html

```diff
@@ -273,6 +275,38 @@ <h2 class="text-3xl font-semibold pb-4 text-cyan-500 tracking-widest" style="fon
             <div class="w-40 h-2 bg-cyan-500 mb-6" style="clip-path: polygon(5px 0px, 100% 0px, 100% 40%, calc(100% - 5px) 100%, 0px 100%, 0px 60%);"></div>
             <p class="pb-4">8-weeks event where students contribute to open-source projects starting on 19 August. Where we will provide trainings on Git, how to collaborate on GitHub, finding a suitable open source project and contribute to them. Certificates will be awarded on completing the program. Enhance your skills and advance your career with this invaluable opportunity!</p>
             <br class="my-10">
+            <!--stats section-->
+          <div class="stats">
+            <div style="font-family: Orbitron;" class="stats-text Current-status"><h1 class="text-cyan-500 text-3xl font-semibold mb-12 tracking-widest">Current Status</h1></div>
+            <div class="row row1">
+              <div class="first total-prs">
+                <h2 style="font-family: Orbitron;" class="text-gray-300 text-3xl font-semibold tracking-widest stats-number">77</h2>
```

Comment:
You don't have to define `font-family` over and over, just define it in parent div and it will be same for all children.
same for `text-gray-300`, `font-semibold`, `tracking-widest`.

Also you can install tailwind css extension on vscode which can help you easier writing classes for tailwind.

Reply:
done